Regression/prepare_file_travis.py

```diff
@@ -71,7 +71,7 @@
               text)
 
 # Use only 2 cores for compiling
-text = re.sub( 'numMakeJobs = \d+', 'numMakeJobs = 2', text )
+text = re.sub( 'numMakeJobs = \d+', 'numMakeJobs = 6', text )
```

Comment:
oh wait, we can compile 6 cores on Azure?

Reply:
Oh no, that was a local change that slipped in.